plugin/entryBase.go

```diff
@@ -55,61 +55,34 @@ func NewEntry(name string) EntryBase {
 	}
 
 	e := EntryBase{
-		entryName:       name,
-		slashReplacerCh: '#',
+		name:          name,
+		slashReplacer: '#',
 	}
 	for op := range e.ttl {
 		e.SetTTLOf(defaultOpCode(op), 15*time.Second)
 	}
 	return e
 }
 
+func (e *EntryBase) partialMetadata() JSONObject {
+	if e.specifiedPartialMetadata != nil {
+		return e.specifiedPartialMetadata
+	}
+	return e.attributes.ToMap()
+}
+
 // ENTRY INTERFACE
 
-// Metadata returns the entry's meta attribute (see plugin.EntryAttributes).
-// Override this if e has additional metadata.
+// Metadata returns the entry's partial metadata. Override this if e has
```

Comment:
It's worth calling out somewhere that it's normal to set partial metadata and override Metadata, and why that's the case.

Reply:
Updated.